src/com/cdd/bao/editor/EditSchema.java

```diff
@@ -1107,6 +1111,50 @@ public void actionEditRedo()
 		rebuildTree();
 		clearSelection();
 	}
+	public void actionEditFind()
+	{
+		SearchSchema.State state = new SearchSchema.State();
+		
+		ButtonType okBtnType = new ButtonType("Find Next", ButtonData.NEXT_FORWARD);
+		ButtonType cancelBtnType = new ButtonType("Cancel", ButtonBar.ButtonData.CANCEL_CLOSE);
+
+		TextInputDialog findDiag = new TextInputDialog();
+		findDiag.getDialogPane().getButtonTypes().setAll(okBtnType, cancelBtnType);
+		findDiag.setTitle("Find group or assignment");
+		findDiag.setHeaderText("Enter text fragment from group or assignment.");
+
+		final Button okBtn = (Button)findDiag.getDialogPane().lookupButton(okBtnType);
+		okBtn.addEventFilter(ActionEvent.ACTION, event ->
+		{
+			// If empty search text, do nothing. Otherwise, initiate search.
```

Comment:
Sort comments don't use punctuation, i.e. start with lower case and omit the period